app/src/main/java/com/stevesoltys/seedvault/ui/storage/BackupStorageViewModel.kt

```diff
@@ -24,6 +24,7 @@ internal class BackupStorageViewModel(
 
     override fun onLocationSet(uri: Uri) {
         val isUsb = saveStorage(uri)
+        settingsManager.forceStorageInitialization()
```

Comment:
Functionally nothing has changed. This call was just renamed and moved here to make more explicit what is happening. Still not totally happy with the approach, but it is an improvement.